graph/schema/_mutations.js

```diff
@@ -10,6 +10,11 @@ type Mutation {
   deletePodcast(id: ID!): Boolean
   createEpisode(input: EpisodeCreationInput): Episode!
   deleteEpisode(id: ID!): Boolean
+  createEpisodeGroup(input: EpisodeGroupModificationInput): EpisodeGroup!
```

Comment:
What are episode groups and why is it required for the app to create them?

Reply:
that is a feature for letting user group their.interested episodes and podcasts in anyway possible